### DISCUSSION TITLE
Basic icons doesn't display offline

### DISCUSSION THREAD

**Discussion by davidrsch** [Appsilon/shiny.fluent#75] [Q&A] [answered]
Hello,
When talking about **basic icons** I mean the check mark in a check box or the x in a close button, I don't know the proper terminology therefore I am sorry for any confusion this may cause. 
As the title says when working offline the **basic icons** doesn't display correctly:
![checkbox](https://user-images.githubusercontent.com/72680874/128947219-876bc5af-a505-4f5b-9b40-74b2f0730bcb.png)
I am guessing this is normal behavior because these **basic icons** are storage online.
I wanna know if there is any solution for this? or if is Appsilon planning on add this **basic icons** to the shiny.fluent package?

**Comment by kamilzyla** [comment, marked as answer]
Hey @davidrsch, thanks for your question! Fluent UI uses a CDN to serve icons by default, and that's why they are not displaying when you are offline. I created issue #84 to track this problem, but I see it as a low priority item (running Shiny apps offline is not a common use case). We are happy to receive pull requests though :slightly_smiling_face: 

I don't see any easy workaround for this. You could try to call `initializeIcons()` yourself and use Shiny to serve the appropriate files, but I'm not sure if that would work.
